.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
As a convention, we add a new line at the end of every file that allows it (EOF) in order to avoid this symbol when checking differences in github: 

![image](https://user-images.githubusercontent.com/43755493/71993653-3b132080-3216-11ea-9213-990b6d1e5567.png)

Just for future reference 😄

Reply:
Sure, this files were auto generaated thou, So I didn't really check them.